### PR TITLE
Add custom battle bus locations to lategame

### PR DESCRIPTION
**Pull request by NotJeppy**

### CODE REVIEW

File: Raider/GameModes/GameModeLateGame.hpp [hunk 1]
```diff
@@ -13,46 +47,73 @@ class GameModeLateGame : public AbstractGameModeBase
...
-        else
-        {
-            LOG_ERROR("SafeZoneIndicator was NULL! Can't set Radius and Center.");
-        }
+        auto SafeZoneCenter = GetRandomBattleBusLocation(); /* TODO: Array of Locations where late game could take place in. (POIs) */
+        // GameMode->SafeZonePhase = 1;
+        // GameMode->SafeZoneLocations = Aircraft->K2_GetActorLocation();
+        // Aircraft->K2_GetActorLocation();
+        /* TArray<FVector> Locations;
+        Locations.Add(FVector(-123778, -112480, 17525));
```
Comment (NotJeppy) [inline Raider/GameModes/GameModeLateGame.hpp:70]: This was my terrible attempt at custom storms. It does not affect anything in game.

**Review by Kyiro — COMMENTED** [review]
The formatting is kinda weird

**Comment by NotJeppy** [comment]
> The formatting is kinda weird

This was the way that samuel set it up originally, I just changed it up a little bit due to the provided storm code doing nothing because SafeZoneIndicator is always null.

**Comment by Kyiro** [comment]
Sorry for keeping this PR waiting for so long

**Pull request merged**